portfolio/src/main/webapp/script.js

```diff
@@ -28,67 +28,145 @@ function addRandomGreeting() {
+                `This design is my current favorite clothing design I've made. The design is inspired by 
+                the brand Bad Monday Apparel. This design features a dove and makes use of text manipulation 
+                to create an appealing, balanced design. `
+            ],
+            [
+                "eas2012",
+                `This is actually the first design I made for Epsilon Alpha Sigma. This design features 
+                our national founding year in arabic. I was inspired by vintage sports jerseys when designing 
+                the numbers. Also, I chose to use arabic both because of our identity as an Arabic sorority and 
+                because of the shape language of the arabic numerals allows for more interesting designs than 2012 
+                would have.`
+            ]
+        ]
+
+    }
+    else if(clickedId == "traditional"){
+        page_heading = [
+            "Traditional Art",
+            `This page is to showcase my favorite traditional art pieces. While I mainly work in traditional art, 
+            a lot of these pieces are outdated as they were made for my AP Art portfolio. Therefore, I am 
+            only choosing to showcase my best work. Hopefully, over the summer I will have more time to do traditional 
+            art and add to this page.`
+        ]
+    }
+    else if(clickedId == "misc"){
+        page_heading = [
+            "Miscellaneous Art",
+            `This page features all the miscellaneous designs that do not fit in the previous three categories. 
+            As a result of their general randomness, most of these designs can be found on my redbubble. Please 
+            make sure to check it out if you like any of these designs! Most of these designs were made in either 
+            Procreate or Adobe Illustrator.`
+        ]
+        img_ids_desc = [
+            [
+                "misc", 
+                `This is my first design I made with Adobe Illustrator. It was made by creating a blend and then 
+                applying filters and effects until I was satisfied with the composition. After creating the intial 
+                swirl, I applied a slight grain to the whole piece to make it look dated. A lot of my subsequent pieces 
+                use a similar effect.`
+            ],
+            [
+                "passionFruit",
+                `This design was made as a represention of how the song Passionfruit by Drake sounds to me. 
+                This piece is meant to be worn as a shirt design and while the design was made with procreate, the 
+                text was done with illustrator.`
+            ]
+        ]
+    }
+
+    var button = document.createElement("button");
+    var buttonText = document.createTextNode("Back");
+    button.appendChild(buttonText);
+    button.className = "back_btn";
+    button.addEventListener('click', function(){ location.reload();}, false);
+    designPage.appendChild(button);
```

Comment:
considering we aren't doing anything dynamic with this button, if there a reason we're creating it via javascript and not just slapping it directly into the HTML?

Reply:
The button would not have any function before the javascript for the rest of the page is called, so I only have it appear after its called.

Reply:
that's fair, although this should all load super quickly anyway, so the timing shouldn't be a huge issue. It's typically better to have all of your desired elements defined in the HTML and just use javascript to dynamically edit them.

Reply:
the rest of the function and elements on the page only loads once you click an element. Here, I'll attach a picture of what the function does. I'm still not sure it would make sense to put the button on the original page.
![Screenshot 2020-05-28 at 4 24 36 PM](https://user-images.githubusercontent.com/52723280/83190028-cf1b5100-a0ff-11ea-81b8-b7a786ab372c.png)
![Screenshot 2020-05-28 at 4 24 44 PM](https://user-images.githubusercontent.com/52723280/83190047-d6daf580-a0ff-11ea-9946-30a36284de4a.png)

Reply:
ahhhh okay I understand. Thanks for showing!